src/components/organisms/Settings/Project/Dataset/hooks.ts

```diff
@@ -27,15 +27,36 @@ export default (projectId: string) => {
   });
 
   const sceneId = sceneData?.scene?.id;
+  const dataSetPerPage = 20;
```

Comment:
dataSetPerPage -> datasetsPerPage

Also, same as mentioned, move outside hooks since it won't change ever.